.goreleaser.docker.yaml

```diff
@@ -164,7 +142,6 @@ docker_manifests:
     name_template: '{{ .Env.DOCKER_REPO }}/pmap-prober:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}'
```

Comment:
I believe you can remove prober from docker_manifests now since we're not shooting for multiarch image.